<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> High throughput modification substrate assay | Jianheng Liu (刘健恒) </title> <meta name="author" content="Jianheng Liu (刘健恒)"> <meta name="description" content="More oligos, please"> <meta name="keywords" content="Bioinformatics, Biotechnology, RNA modifications"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/fox.ico?b38cd7460e1daba6437e5bc5ad629422"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://jhfoxliu.github.io/projects/published_5_substarte_pools/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Jianheng Liu </span> (刘健恒) </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/fun/">fun</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/tutorials/">tutorials</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/gallery/">gallery</a> </div> </li> <li class="nav-item active"> <a class="nav-link" href="/projects/">projects <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">High throughput modification substrate assay</h1> <p class="post-description">More oligos, please</p> </header> <article> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-1"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/mutation_assay/mutation_assay_exp-480.webp"></source> <source media="(max-width: 800px)" srcset="/assets/img/mutation_assay/mutation_assay_exp-800.webp"></source> <source media="(max-width: 1400px)" srcset="/assets/img/mutation_assay/mutation_assay_exp-1400.webp"></source> <img class="img-fluid rounded z-depth-1" src="/assets/img/mutation_assay/mutation_assay_exp.jpg" data-zoomable=""> </picture> </figure> </div> </div> <div class="caption"> The workflow of high-throughput mutation assay: we synthesized the oligos, inserted them into a vector, purify the library, transfect them into the cells, and get reporter methylation levels from sequencing. </div> <p>This is not a new strategy. In a <a href="https://pubmed.ncbi.nlm.nih.gov/28073919/" rel="external nofollow noopener" target="_blank">previous paper</a> studying pseudouridine, they adopt a same strategy to draw the consensus modification substrate of TRUB1. Here, we used pools to analyze the substrate recognition patterns of NSUN2 and NSUN6. A brief summary:</p> <ul> <li>NSUN2 <ul> <li>NSUN2 requires a stable stem loop in the 3’ downstream adjacent to the m5C sites.</li> <li>More Gs in the stem loop will result in higher methylation levels, but G is not so necessary for methylation in some cases.</li> </ul> </li> <li>NSUN6 <ul> <li>NSUN6 is very sensitive to the change in the core motif (Nm5CUCCA).</li> <li>NSUN6 can tolerate some +3 C-to-U changes.</li> <li>The mechanism of preferences in NSUN6-mRNA recognition can be explained by our structure model.</li> </ul> </li> </ul> <p><br></p> <h4 id="references">References</h4> <div class="publications"> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Nat. Commun.</abbr> </div> <div id="liu2022nc" class="col-sm-8"> <div class="title">[accpeted] Developmental mRNA m5C landscape and regulatory innovations of massive m5C modification of maternal mRNAs in animals</div> <div class="author"> <em>Jianheng Liu</em> ,  Tao Huang ,  Wanying Chen , and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Chenhui Ding, Tianxuan Zhao, Xueni Zhao, Bing Cai, Yusen Zhang, Song Li, Ling Zhang, Maoguang Xue, Xiuju He, Wanzhong Ge, Canquan Zhou, Yanwen Xu, Rui Zhang' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>m5C is one of the longest-known RNA modifications, however, its developmental dynamics, functions, and evolution in mRNAs remain largely unknown. Here, we generate quantitative mRNA m5C maps at different stages of development in 6 vertebrate and invertebrate species and find convergent and unexpected massive methylation of maternal mRNAs mediated by NSUN2 and NSUN6. Using Drosophila as a model, we reveal that embryos lacking maternal mRNA m5C undergo cell cycle delays and fail to timely initiate maternal-to-zygotic transition, implying the functional importance of maternal mRNA m5C. From invertebrates to the lineage leading to humans, two waves of m5C regulatory innovations are observed: higher animals gain cis-directed NSUN2-mediated m5C sites at the 5’ end of the mRNAs, accompanied by the emergence of more structured 5’UTR regions; humans gain thousands of trans-directed NSUN6-mediated m5C sites enriched in genes regulating the mitotic cell cycle. Collectively, our studies highlight the existence and regulatory innovations of a mechanism of early embryonic development and provide key resources for elucidating the role of mRNA m5C in biology and disease.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">NSR</abbr> </div> <div id="liu2021sequence" class="col-sm-8"> <div class="title">Sequence-and structure-selective mRNA m5C methylation by NSUN6 in animals</div> <div class="author"> <em>Jianheng Liu</em> ,  Tao Huang ,  Yusen Zhang , and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Tianxuan Zhao, Xueni Zhao, Wanying Chen, Rui Zhang' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>National Science Review</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://academic.oup.com/nsr/article/8/6/nwaa273/5948103" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/NSR_2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>mRNA m5C, which has recently been implicated in the regulation of mRNA mobility, metabolism and translation, plays important regulatory roles in various biological events. Two types of m5C sites are found in mRNAs. Type I m5C sites, which contain a downstream G-rich triplet motif and are computationally predicted to be located at the 5’ end of putative hairpin structures, are methylated by NSUN2. Type II m5C sites contain a downstream UCCA motif and are computationally predicted to be located in the loops of putative hairpin structures. However, their biogenesis remains unknown. Here we identified NSUN6, a methyltransferase that is known tomethylate C72 of tRNAThr and tRNA=Cys, as an mRNAmethyltransferase that targets Type II m5C sites. Combining the RNA secondary structure prediction, miCLIP, and results from a high-throughput mutagenesis analysis, we determined the RNA sequence and structural features governing the specificity of NSUN6-mediated mRNA methylation. Integrating these features into an NSUN6-RNA structural model, we identified an NSUN6 variant that largely loses tRNA methylation but retains mRNA methylation ability. Finally, we revealed a weak negative correlation between m5C methylation and translation efficiency. Our findings uncover that mRNA m5C is tightly controlled by an elaborate two-enzyme system, and the protein-RNA structure analysis strategy established may be applied to other RNA modification writers to distinguish the functions of different RNA substrates of a writer protein.</p> </div> </div> </div> </li></ol> </div> <p><br></p> <p><br> <a href="/projects/"><u>Back</u></a></p> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Jianheng Liu (刘健恒). Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>