<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Jianheng Liu (刘健恒) </title> <meta name="author" content="Jianheng Liu (刘健恒)"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="Bioinformatics, Biotechnology, RNA modifications"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/fox.ico?b38cd7460e1daba6437e5bc5ad629422"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://jhfoxliu.github.io/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about <span class="sr-only">(current)</span> </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/fun/">fun</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/tutorials/">tutorials</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/gallery/">gallery</a> </div> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Jianheng Liu</span> (刘健恒) </h1> <p class="desc">Postdoc</p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/prof_pic_2-480.webp 480w,/assets/img/prof_pic_2-800.webp 800w,/assets/img/prof_pic_2-1400.webp 1400w," sizes="(min-width: 800px) 231.0px, (min-width: 576px) 30vw, 95vw" type="image/webp"> <img src="/assets/img/prof_pic_2.jpg?57aa939a19c638dff118846fc3851dd4" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic_2.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> </div> <div class="clearfix"> <p>Welcome to my website!</p> <p>You can also call me <a href="https://en.wikipedia.org/wiki/Fox_Mulder" rel="external nofollow noopener" target="_blank"><code class="language-plaintext highlighter-rouge">Fox</code></a>.</p> <p>I am a <code class="language-plaintext highlighter-rouge">full-stack</code> biology geek experienced in biotechnology and bioinformatics. My ambition is to build revolutionary paradigms to increase our understanding about the biological world. To achieve this, I would love to apply any available approaches. I also love creating beautiful <a href="/gallery/">figures and illustrations</a>, which can help us broadcast the fantastic findings. I am also writing some <a href="/tutorials/">tutorials</a> on the website.</p> <p>I was graduated from <a href="https://lifesciences.sysu.edu.cn/teachers/professor" rel="external nofollow noopener" target="_blank">Rui Zhang’s Lab</a> @ Sun Yat-sen Univerisity and I am now moving to <a href="https://jaffreylab.org/" rel="external nofollow noopener" target="_blank">Jaffrey’s lab</a> for my postodc career. If you are interested in my stories, click <a href="https://jhfoxliu.github.io/blog/category/story/">here</a>.</p> <p>And <a href="/assets/pdf/cv/liujh_cv.pdf">here</a> is my Curriculum Vitae. (not up-to-date)</p> <p><a href="/fun/">Besides work</a>, I love games, photograph, animations, and observing the nature.</p> <p>I also love trying tasty foods and I love making them by myself (my hometown <a href="https://en.wikipedia.org/wiki/Shunde_District" rel="external nofollow noopener" target="_blank">Shunde</a> is an amazing place for foods!).</p> </div> <h2> <a href="/news/" style="color: inherit">news</a> </h2> <div class="news"> <div class="table-responsive" style="max-height: 60vw"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Jun 27, 2022</th> <td> Hi guys, I am moving to <a href="https://jaffreylab.org/" rel="external nofollow noopener" target="_blank">Jaffrey’s Lab</a> at Weill Cornell Medical College! See you in New York! </td> </tr> <tr> <th scope="row" style="width: 20%">May 05, 2022</th> <td> My Nature Communications paper is now online! <a href="https://www.nature.com/articles/s41467-022-30210-0" rel="external nofollow noopener" target="_blank">(link)</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Mar 13, 2022</th> <td> <a class="news-title" href="/news/announcement_5/">WARNING, a 16-17 years old male cat is fertile</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Mar 10, 2022</th> <td> I release the iMVP project on Github. Click <a href="https://github.com/jhfoxliu/iMVP" rel="external nofollow noopener" target="_blank">here</a> for details. </td> </tr> <tr> <th scope="row" style="width: 20%">Mar 03, 2022</th> <td> I am deploying this website… </td> </tr> </table> </div> </div> <h2> <a href="/publications/" style="color: inherit">selected publications</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Nat. Commun.</abbr> </div> <div id="liu2023epitranscriptomic" class="col-sm-8"> <div class="title">Epitranscriptomic subtyping, visualization, and denoising by global motif visualization</div> <div class="author"> <em>Jianheng Liu</em> ,  Tao Huang ,  Jing Yao , and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Tianxuan Zhao, Yusen Zhang, Rui Zhang' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Nature Communications</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41467-023-41653-4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/NC_2023.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Advances in sequencing technologies have empowered epitranscriptomic profiling at the single-base resolution. Putative RNA modification sites identified from a single high-throughput experiment may contain one type of modification deposited by different writers or different types of modifications, along with false positive results because of the challenge of distinguishing signals from noise. However, current tools are insufficient for subtyping, visualization, and denoising these signals. Here, we present iMVP, which is an interactive framework for epitranscriptomic analysis with a nonlinear dimension reduction technique and density-based partition. As exemplified by the analysis of mRNA m5C and ModTect variant data, we show that iMVP allows the identification of previously unknown RNA modification motifs and writers and the discovery of false positives that are undetectable by traditional methods. Using putative m6A/m6Am sites called from 8 profiling approaches, we illustrate that iMVP enables comprehensive comparison of different approaches and advances our understanding of the difference and pattern of true positives and artifacts in these methods. Finally, we demonstrate the ability of iMVP to analyze an extremely large human A-to-I editing dataset that was previously unmanageable. Our work provides a general framework for the visualization and interpretation of epitranscriptomic data.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Nat. Commun.</abbr> </div> <div id="liu2022nc" class="col-sm-8"> <div class="title">Developmental mRNA m5C landscape and regulatory innovations of massive m5C modification of maternal mRNAs in animals</div> <div class="author"> <em>Jianheng Liu</em> ,  Tao Huang ,  Wanying Chen , and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Chenhui Ding, Tianxuan Zhao, Xueni Zhao, Bing Cai, Yusen Zhang, Song Li, Ling Zhang, Maoguang Xue, Xiuju He, Wanzhong Ge, Canquan Zhou, Yanwen Xu, Rui Zhang' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>Nature Communications</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41467-022-30210-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/NC_2022.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>m5C is one of the longest-known RNA modifications, however, its developmental dynamics, functions, and evolution in mRNAs remain largely unknown. Here, we generate quantitative mRNA m5C maps at different stages of development in 6 vertebrate and invertebrate species and find convergent and unexpected massive methylation of maternal mRNAs mediated by NSUN2 and NSUN6. Using Drosophila as a model, we reveal that embryos lacking maternal mRNA m5C undergo cell cycle delays and fail to timely initiate maternal-to-zygotic transition, implying the functional importance of maternal mRNA m5C. From invertebrates to the lineage leading to humans, two waves of m5C regulatory innovations are observed: higher animals gain cis-directed NSUN2-mediated m5C sites at the 5’ end of the mRNAs, accompanied by the emergence of more structured 5’UTR regions; humans gain thousands of trans-directed NSUN6-mediated m5C sites enriched in genes regulating the mitotic cell cycle. Collectively, our studies highlight the existence and regulatory innovations of a mechanism of early embryonic development and provide key resources for elucidating the role of mRNA m5C in biology and disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">NSR</abbr> </div> <div id="liu2021sequence" class="col-sm-8"> <div class="title">Sequence-and structure-selective mRNA m5C methylation by NSUN6 in animals</div> <div class="author"> <em>Jianheng Liu</em> ,  Tao Huang ,  Yusen Zhang , and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Tianxuan Zhao, Xueni Zhao, Wanying Chen, Rui Zhang' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>National Science Review</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://academic.oup.com/nsr/article/8/6/nwaa273/5948103" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/NSR_2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>mRNA m5C, which has recently been implicated in the regulation of mRNA mobility, metabolism and translation, plays important regulatory roles in various biological events. Two types of m5C sites are found in mRNAs. Type I m5C sites, which contain a downstream G-rich triplet motif and are computationally predicted to be located at the 5’ end of putative hairpin structures, are methylated by NSUN2. Type II m5C sites contain a downstream UCCA motif and are computationally predicted to be located in the loops of putative hairpin structures. However, their biogenesis remains unknown. Here we identified NSUN6, a methyltransferase that is known tomethylate C72 of tRNAThr and tRNA=Cys, as an mRNAmethyltransferase that targets Type II m5C sites. Combining the RNA secondary structure prediction, miCLIP, and results from a high-throughput mutagenesis analysis, we determined the RNA sequence and structural features governing the specificity of NSUN6-mediated mRNA methylation. Integrating these features into an NSUN6-RNA structural model, we identified an NSUN6 variant that largely loses tRNA methylation but retains mRNA methylation ability. Finally, we revealed a weak negative correlation between m5C methylation and translation efficiency. Our findings uncover that mRNA m5C is tightly controlled by an elaborate two-enzyme system, and the protein-RNA structure analysis strategy established may be applied to other RNA modification writers to distinguish the functions of different RNA substrates of a writer protein.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">NSMB</abbr> </div> <div id="huang2019genome" class="col-sm-8"> <div class="title">Genome-wide identification of mRNA 5-methylcytosine in mammals</div> <div class="author"> Tao Huang ,  Wanying Chen ,  <em>Jianheng Liu</em> , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Nannan Gu, Rui Zhang' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Nature structural &amp; molecular biology</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41594-019-0218-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/NSMB_2019.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Accurate and systematic transcriptome-wide detection of 5-methylcytosine (m5C) has proved challenging, and there are conflicting views about the prevalence of this modification in mRNAs. Here we report an experimental and computational framework that robustly identified mRNA m5C sites and determined sequence motifs and structural features associated with the modification using a set of high-confidence sites. We developed a quantitative atlas of RNA m5C sites in human and mouse tissues based on our framework. In a given tissue, we typically identified several hundred exonic m5C sites. About 62-70% of the sites had low methylation levels (20% methylation), while 8-10% of the sites were moderately or highly methylated (40% methylation). Cross-species analysis revealed that species, rather than tissue type, was the primary determinant of methylation levels, indicating strong cis-directed regulation of RNA methylation. Combined, these data provide a valuable resource for identifying the regulation and functions of RNA methylation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Genome Res.</abbr> </div> <div id="li2018landscape" class="col-sm-8"> <div class="title">The landscape of miRNA editing in animals and its impact on miRNA biogenesis and targeting</div> <div class="author"> Lishi Li ,  Yulong Song ,  Xinrui Shi , and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Jianheng Liu, Shaolei Xiong, Wanying Chen, Qiang Fu, Zichao Huang, Nannan Gu, Rui Zhang' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Genome research</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5749178/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/GR_2018.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Adenosine-to-inosine (A-to-I) RNA editing regulates miRNA biogenesis and function. To date, fewer than 160 miRNA editing sites have been identified. Here, we present a quantitative atlas of miRNA A-to-I editing through the profiling of 201 pri-miRNA samples and 4694 mature miRNA samples in human, mouse, and Drosophila. We identified 4162 sites present in  80% of the pri-miRNAs and 574 sites in mature miRNAs. miRNA editing is prevalent in many tissue types in human. However, high-level editing is mostly found in neuronal tissues in mouse and Drosophila. Interestingly, the edited miRNAs in neuronal and non-neuronal tissues in human gain two distinct sets of new targets, which are significantly associated with cognitive and organ developmental functions, respectively. Furthermore, we reveal that miRNA editing profoundly affects asymmetric strand selection. Altogether, these data provide insight into the impact of RNA editing on miRNA biology and suggest that miRNA editing has recently gained non-neuronal functions in human.</p> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%6A%68%66%6F%78%6C%69%75@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://orcid.org/0000-0003-0216-1951" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=jhfoxliu@gmail.com" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://www.researchgate.net/profile/Jianheng-Liu/" title="ResearchGate" rel="external nofollow noopener" target="_blank"><i class="ai ai-researchgate"></i></a> <a href="https://github.com/jhfoxliu" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> <a href="https://www.linkedin.com/in/jianheng-liu-79b1aa1b5" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-linkedin"></i></a> <a href="/feed.xml" title="RSS Feed"><i class="fa-solid fa-square-rss"></i></a> </div> <div class="contact-note">Email jhfoxliu@gmail.com and I will response to you soon. </div> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Jianheng Liu (刘健恒). Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>