<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> mRNA BS-seq pipeline and NSUN2 | Jianheng Liu (刘健恒) </title> <meta name="author" content="Jianheng Liu (刘健恒)"> <meta name="description" content="We still don't know the function and mechanism behind it"> <meta name="keywords" content="Bioinformatics, Biotechnology, RNA modifications"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/fox.ico?b38cd7460e1daba6437e5bc5ad629422"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://jhfoxliu.github.io/projects/published_3_BS-seq/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Jianheng Liu </span> (刘健恒) </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/fun/">fun</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/tutorials/">tutorials</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/gallery/">gallery</a> </div> </li> <li class="nav-item active"> <a class="nav-link" href="/projects/">projects <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">mRNA BS-seq pipeline and NSUN2</h1> <p class="post-description">We still don't know the function and mechanism behind it</p> </header> <article> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/m5C_pipeline/NSMB-480.webp"></source> <source media="(max-width: 800px)" srcset="/assets/img/m5C_pipeline/NSMB-800.webp"></source> <source media="(max-width: 1400px)" srcset="/assets/img/m5C_pipeline/NSMB-1400.webp"></source> <img class="img-fluid rounded z-depth-1" src="/assets/img/m5C_pipeline/NSMB.jpg" data-zoomable=""> </picture> </figure> </div> </div> <div class="caption"> The schema of our optimized workflow. </div> <h3 id="in-breif">In breif</h3> <ul> <li>We optimized the experimental protocol to improve the conversion rates in mRNA BS-seq.</li> <li>We analyzed the pattern of artifacts in BS-seq and located the source of false positives.</li> <li>We introduced <code class="language-plaintext highlighter-rouge">Gini-index</code> and <code class="language-plaintext highlighter-rouge">signal-nosie ratio</code> into analysis to estimate the status of library conversion.</li> <li>We introduced <code class="language-plaintext highlighter-rouge">C-cutoff</code> and <code class="language-plaintext highlighter-rouge">gene-specific conversion rates</code> into our filter to remove false positives.</li> <li>We set up a <a href="https://github.com/jhfoxliu/RNA-m5C" rel="external nofollow noopener" target="_blank"><code class="language-plaintext highlighter-rouge">new pipeline</code></a> with high flexibility, robustness and parallelization level for mRNA BS-seq.</li> <li>We validated that NSUN2 is one of the major m5C methyltransferase acting on mRNA.</li> <li>We profile mRNA landscapes in multiple human/mouse tissues.</li> <li>mRNA m5C might relate to translation control.</li> </ul> <p><br></p> <h3 id="where-are-the-noise">Where are the noise?</h3> <p>It’s clear now that the noise in BS-seq come from many places:</p> <ol> <li>The human-introduced Cs in hexamer.</li> <li>The regions hard to be converted, especially some highly structured rRNA regions.</li> <li>The false alignments and bases with bad sequencing qualities.</li> </ol> <p><br></p> <h3 id="solutions">Solutions</h3> <p>The pipeline I designed have considered most of the situations:</p> <ol> <li>Hexamer can be removed by trimming the bases at the end of reads.</li> <li>We should use <code class="language-plaintext highlighter-rouge">gene-specific</code> conversion rates rather than a <code class="language-plaintext highlighter-rouge">global conversion rate</code> measured by spike-ins or the whole libray. Only analyze annotated regions.</li> <li>Estimate the conversion status by <code class="language-plaintext highlighter-rouge">Gini-index</code> and <code class="language-plaintext highlighter-rouge">signal-nosie ratio</code>, and determine the <code class="language-plaintext highlighter-rouge">C-cutoff</code> that remove reads have more than a certain number of reads.</li> <li>Optimize the alignment step to reduce false alignments. Normally, that should not be a matter.</li> <li>SNPs can only <strong>destory</strong> m5C, but cannot create m5C in BS-seq.</li> <li>Statistical tests have little effect on our analysis if we use a filter with high coverage and level.</li> </ol> <p>In my hands, I guess we can let the sites with 5% methylation go, but we normally use a 10% cutoff for the biological importance of a site. BS-seq is very sensitive and reliable if you work with it correctly.</p> <p><br></p> <div class="row mt-3"> <div class="col-sm mt-2 mt-md-0"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/m5C_pipeline/GeneSpecific%20conversion-480.webp"></source> <source media="(max-width: 800px)" srcset="/assets/img/m5C_pipeline/GeneSpecific%20conversion-800.webp"></source> <source media="(max-width: 1400px)" srcset="/assets/img/m5C_pipeline/GeneSpecific%20conversion-1400.webp"></source> <img class="img-fluid rounded z-depth-1" src="/assets/img/m5C_pipeline/GeneSpecific%20conversion.jpg" data-zoomable=""> </picture> </figure> </div> </div> <div class="col-sm mt-3 mt-md-0"> <div class="row mt-1"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/m5C_pipeline/gini-480.webp"></source> <source media="(max-width: 800px)" srcset="/assets/img/m5C_pipeline/gini-800.webp"></source> <source media="(max-width: 1400px)" srcset="/assets/img/m5C_pipeline/gini-1400.webp"></source> <img class="img-fluid rounded z-depth-1" src="/assets/img/m5C_pipeline/gini.jpg" data-zoomable=""> </picture> </figure> </div> </div> <div class="row mt-1"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/m5C_pipeline/C-cutoff_signal-480.webp"></source> <source media="(max-width: 800px)" srcset="/assets/img/m5C_pipeline/C-cutoff_signal-800.webp"></source> <source media="(max-width: 1400px)" srcset="/assets/img/m5C_pipeline/C-cutoff_signal-1400.webp"></source> <img class="img-fluid rounded z-depth-1" src="/assets/img/m5C_pipeline/C-cutoff_signal.jpg" data-zoomable=""> </picture> </figure> </div> </div> </div> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/m5C_pipeline/steps-480.webp"></source> <source media="(max-width: 800px)" srcset="/assets/img/m5C_pipeline/steps-800.webp"></source> <source media="(max-width: 1400px)" srcset="/assets/img/m5C_pipeline/steps-1400.webp"></source> <img class="img-fluid rounded z-depth-1" src="/assets/img/m5C_pipeline/steps.jpg" data-zoomable=""> </picture> </figure> </div> </div> <div class="caption"> left, why gene-specific conversion rates are important; middle, gini-index vs signal rates in badly converted samples and well converted samples. And some examples for true positives and false positives; right, our filters can work perfectly! </div> <p><br></p> <h4 id="references">References</h4> <div class="publications"> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">NSMB</abbr> </div> <div id="huang2019genome" class="col-sm-8"> <div class="title">Genome-wide identification of mRNA 5-methylcytosine in mammals</div> <div class="author"> Tao Huang ,  Wanying Chen ,  <em>Jianheng Liu</em> , and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Nannan Gu, Rui Zhang' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Nature structural &amp; molecular biology</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.nature.com/articles/s41594-019-0218-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/NSMB_2019.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Accurate and systematic transcriptome-wide detection of 5-methylcytosine (m5C) has proved challenging, and there are conflicting views about the prevalence of this modification in mRNAs. Here we report an experimental and computational framework that robustly identified mRNA m5C sites and determined sequence motifs and structural features associated with the modification using a set of high-confidence sites. We developed a quantitative atlas of RNA m5C sites in human and mouse tissues based on our framework. In a given tissue, we typically identified several hundred exonic m5C sites. About 62-70% of the sites had low methylation levels (20% methylation), while 8-10% of the sites were moderately or highly methylated (40% methylation). Cross-species analysis revealed that species, rather than tissue type, was the primary determinant of methylation levels, indicating strong cis-directed regulation of RNA methylation. Combined, these data provide a valuable resource for identifying the regulation and functions of RNA methylation.</p> </div> </div> </div> </li></ol> </div> <p><br></p> <p><br> <a href="/projects/"><u>Back</u></a></p> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Jianheng Liu (刘健恒). Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>