<!doctype html> <html lang="en"> <head> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> The NSUN6 poject | Jianheng Liu (刘健恒) </title> <meta name="author" content="Jianheng Liu (刘健恒)"> <meta name="description" content="Traditional molecular biolog plus simulations"> <meta name="keywords" content="Bioinformatics, Biotechnology, RNA modifications"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/fox.ico?b38cd7460e1daba6437e5bc5ad629422"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://jhfoxliu.github.io/projects/published_2_NSUN6/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Jianheng Liu&nbsp;</span> (刘健恒) </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/fun/">fun</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/tutorials/">tutorials</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/gallery/">gallery</a> </div> </li> <li class="nav-item active"> <a class="nav-link" href="/projects/">projects <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">The NSUN6 poject</h1> <p class="post-description">Traditional molecular biolog plus simulations</p> </header> <article> <h3 id="in-breif">In breif</h3> <ul> <li>We verified that NSUN6 is a key contributor to mRNA m5C.</li> <li>NSUN6 favors a 5’-m5CUCCA-3’ motif and usually methylates cytosines in a small loop adjacent to a longer stem.</li> <li>NSUN6 has different activity in different types of tissues/cells.</li> <li>NSUN6 methylates mRNA in the cytoplasm, while NSUN2 methylates mRNA in both nucleus and cytoplasm.</li> <li>We profile the m5C in polysome fractions, and we found that m5C enriched in fractions with fewer ribosomes.</li> <li>We use simulations (RBP-denovo) to study the interactions between NSUN6 and mRNA.</li> <li>Based on the structure model, we try to mutate some residues to investigate the feature of the NSUN6-mRNA complex.</li> <li>We use high-throughput <a href="/projects/published_5_substarte_pools/"><code class="language-plaintext highlighter-rouge">mutation asssay</code></a> to study the substrate recognition of NSUN6.</li> <li>We design a new <a href="https://github.com/jhfoxliu/tRNA-m5C"><code class="language-plaintext highlighter-rouge">tRNA BS-seq pipeline</code></a> to study the changes of m5C levels on tRNA and mRNA.</li> </ul> <p><br/></p> <h3 id="about-rbp-structure-simulation">About RBP structure simulation</h3> <div class="row mt-3"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/NSUN6/simulations_1-480.webp"/> <source media="(max-width: 800px)" srcset="/assets/img/NSUN6/simulations_1-800.webp"/> <source media="(max-width: 1400px)" srcset="/assets/img/NSUN6/simulations_1-1400.webp"/> <img class="img-fluid rounded z-depth-1" src="/assets/img/NSUN6/simulations_1.jpg" data-zoomable=""/> </picture> </figure> </div> <div class="col-sm mt-3 mt-md-1"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/NSUN6/Homology-480.webp"/> <source media="(max-width: 800px)" srcset="/assets/img/NSUN6/Homology-800.webp"/> <source media="(max-width: 1400px)" srcset="/assets/img/NSUN6/Homology-1400.webp"/> <img class="img-fluid rounded z-depth-1" src="/assets/img/NSUN6/Homology.jpg" data-zoomable=""/> </picture> </figure> </div> <div class="col-sm mt-3 mt-md-2"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/NSUN6/simulations_2-480.webp"/> <source media="(max-width: 800px)" srcset="/assets/img/NSUN6/simulations_2-800.webp"/> <source media="(max-width: 1400px)" srcset="/assets/img/NSUN6/simulations_2-1400.webp"/> <img class="img-fluid rounded z-depth-1" src="/assets/img/NSUN6/simulations_2.jpg" data-zoomable=""/> </picture> </figure> </div> </div> <div class="caption"> left, simulated model for NSUN6-RNA complex; center, the homology models of NSUN6; right, the homology modelling results explaining the preference of NSUN6 substrate recognition (details see my publication: NSR 2021) </div> <p>For a long time, I have a concern about our technique of <code class="language-plaintext highlighter-rouge">simulation</code> in biological studies: we can use <code class="language-plaintext highlighter-rouge">simulation</code> to boost our design in experiments, but how can we use <code class="language-plaintext highlighter-rouge">simulation</code> to directly understand our nature? An obvious thing is, if the object is hard to be finished/validated via <code class="language-plaintext highlighter-rouge">experiemnts</code>, so how can we know we are doing the correct things with our algroithms running in our computers? If the object is simple in reality, what is the meaning of <code class="language-plaintext highlighter-rouge">simulation</code> itself?</p> <p>In 2020, the 14 th Critical Assessment of protein Structure Prediction (CASP), <code class="language-plaintext highlighter-rouge">AlphaFold</code> exhibited its fantastic performances in the test. It seems that if the RMS of the predition were high enough (&gt;90%), the preditions can be treated as native structures. I was once stimulated, but quickly depressed by the structures of some big, unknown domains.</p> <p>Back to this poject. This project is launched much early than CASP 14th in 2019, when <a href="https://www.sciencedirect.com/science/article/pii/S0969212618303629"><code class="language-plaintext highlighter-rouge">RNP-denovo</code></a> was just published one year ago. And luckly, we have the structure of the <a href="https://www.rcsb.org/structure/5WWS"><code class="language-plaintext highlighter-rouge">NSUN6-mRNA complex</code></a>. It’s not an accidence for me to combine these two things. I have once be a player of <a href="https://fold.it/"><code class="language-plaintext highlighter-rouge">Foldit</code></a> and I am a long-term compuational contributor to <a href="https://boinc.bakerlab.org/"><code class="language-plaintext highlighter-rouge">Rosetta@home</code></a>. I have been thinking about how to use the amazing utilities in my research. Here is the chance.</p> <p>Now, I would like to set up a paradigm: if we have a structure model of RNA binding protein structure and if we have a reference structure of the RNA sequence (not so necessary), we can simulate the complex of RBP-RNA complex, and this will help us in learning the process and preference of RBP-substrate recognition. More importantly, we can be inspired to modify some residues to make the RBP different. In this project, what I want to do is to create a NSUN6 mutant that can only methylate mRNA or tRNA. This purpose comes from the challenge in RNA modification studies, we are unable to distinguish the functions of a modification writer in three aspects: RNA binding, RNA modification itself (a sub-issue is the different types of RNA), and others. <sub>We are annoyed that the editors and reviewers always ask for the function of the modifications, but to be purdent, it’s too hard for us to accurately locate the function without any interference, why others can pubish casualy?</sub> We finally generated a NSUN6 variant that can methylate mRNAs normally, but have a weaker methylation activity on tRNAs. Standard structural analysis also performed to learn the preference of NSUN6.</p> <div class="row mt-3"> <div class="col-sm mt-1 mt-md-0"> </div> <div class="col-sm mt-1 mt-md-0 text-center"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/NSUN6/NSUN6_variants.PNG-480.webp"/> <source media="(max-width: 800px)" srcset="/assets/img/NSUN6/NSUN6_variants.PNG-800.webp"/> <source media="(max-width: 1400px)" srcset="/assets/img/NSUN6/NSUN6_variants.PNG-1400.webp"/> <img class="img-fluid rounded z-depth-1" src="/assets/img/NSUN6/NSUN6_variants.PNG" data-zoomable=""/> </picture> </figure> </div> <div class="col-sm mt-1 mt-md-0"> </div> </div> <div class="caption"> The K159A/R181A mutant can be used for further research. </div> <p>However, this paradigm is still limited in practices:</p> <ol> <li> <p>High-quality protein models, in complex or in apo state, are available in most of the cases. And for some proteins, the apo state is different from that with a substrate.</p> </li> <li> <p>We tried to build up the simulated models for protein with missing reference structure. However, the current simulation tools, include <code class="language-plaintext highlighter-rouge">Rosetta</code> and <code class="language-plaintext highlighter-rouge">AlphaFold</code>, are unable to draw structures without any prior knowledge. For example, the C-termial of human NSUN2 protein has no homology structure available in our database, and hence both <code class="language-plaintext highlighter-rouge">Rosetta</code> and <code class="language-plaintext highlighter-rouge">AlphaFold</code> failed to simulate that part of the protein.</p> </li> <li> <p>We still don’t have a complete theroy to guide us how to adjust parameters, and how to select the right model for further simulation. <sub> And bugs in the softwares… </sub></p> </li> </ol> <div class="row"> <div class="col-sm mt-1 mt-md-0 text-center"> </div> <div class="col-sm mt-1 mt-md-0 text-center"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/NSUN6/vs_alphafold-480.webp"/> <source media="(max-width: 800px)" srcset="/assets/img/NSUN6/vs_alphafold-800.webp"/> <source media="(max-width: 1400px)" srcset="/assets/img/NSUN6/vs_alphafold-1400.webp"/> <img class="img-fluid rounded z-depth-1 mw-50" src="/assets/img/NSUN6/vs_alphafold.jpg" data-zoomable=""/> </picture> </figure> </div> <div class="col-sm mt-1 mt-md-0 text-center"> </div> </div> <div class="caption"> Structures: known homology structure of TRM4 (magenta), predicted NSUN2 structure from Rosetta web sever (green), and predicted NSUN2 structure from AlphaFold (yellow). </div> <p><br/></p> <h4 id="references">References</h4> <div class="publications"> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li><div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">NSR</abbr> </div> <div id="liu2021sequence" class="col-sm-8"> <div class="title">Sequence-and structure-selective mRNA m5C methylation by NSUN6 in animals</div> <div class="author"> <em>Jianheng Liu</em> ,&nbsp; Tao Huang ,&nbsp; Yusen Zhang , and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Tianxuan Zhao, Xueni Zhao, Wanying Chen, Rui Zhang' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>National Science Review</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://academic.oup.com/nsr/article/8/6/nwaa273/5948103" class="btn btn-sm z-depth-0" role="button">HTML</a> <a href="/assets/pdf/NSR_2021.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>mRNA m5C, which has recently been implicated in the regulation of mRNA mobility, metabolism and translation, plays important regulatory roles in various biological events. Two types of m5C sites are found in mRNAs. Type I m5C sites, which contain a downstream G-rich triplet motif and are computationally predicted to be located at the 5’ end of putative hairpin structures, are methylated by NSUN2. Type II m5C sites contain a downstream UCCA motif and are computationally predicted to be located in the loops of putative hairpin structures. However, their biogenesis remains unknown. Here we identified NSUN6, a methyltransferase that is known tomethylate C72 of tRNAThr and tRNA=Cys, as an mRNAmethyltransferase that targets Type II m5C sites. Combining the RNA secondary structure prediction, miCLIP, and results from a high-throughput mutagenesis analysis, we determined the RNA sequence and structural features governing the specificity of NSUN6-mediated mRNA methylation. Integrating these features into an NSUN6-RNA structural model, we identified an NSUN6 variant that largely loses tRNA methylation but retains mRNA methylation ability. Finally, we revealed a weak negative correlation between m5C methylation and translation efficiency. Our findings uncover that mRNA m5C is tightly controlled by an elaborate two-enzyme system, and the protein-RNA structure analysis strategy established may be applied to other RNA modification writers to distinguish the functions of different RNA substrates of a writer protein.</p> </div> </div> </div> </li></ol> </div> <p><br/></p> <p><br/> <a href="/projects/"><u>Back</u></a></p> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> &copy; Copyright 2024 Jianheng Liu (刘健恒). Powered by <a href="https://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>